test/test_job.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from pytest import fixture
+import pytest
+import vcr
+
+from marquez_client.marquez import MarquezClient
+
+
+@fixture(scope='class')
+def set_valid_env():
+    os.environ[MarquezClient.MARQUEZ_HOST_KEY] = "localhost"
+    os.environ[MarquezClient.MARQUEZ_PORT_KEY] = "8080"
+
+
+@fixture(scope='class')
+def marquez_client(set_valid_env):
+    return MarquezClient()
+
+
+@fixture(scope='class')
+@vcr.use_cassette('fixtures/vcr/test_job/namespace_for_job.yaml')
+def namespace(marquez_client):
```

Comment:
I don't see this method called elsewhere

Reply:
It's a test fixture - used in `test_create_job(..)`